src/actions/playground.ts

```diff
@@ -0,0 +1,27 @@
+import { Action, ActionCreator } from 'redux'
+
+/**
+ * The `type` attribute for an `Action` which updates the `IPlaygroundState`
+ * `editorValue`
+ */
+export const UPDATE_EDITOR_VALUE: string = 'UPDATE_EDITOR_VALUE'
+
+/**
+ * Represents an `Action` which updates the `editorValue` of a
+ * `IPlaygroundState`
+ * @property type           - Unique string identifier for this `Action`
+ * @property newEditorValue - The new string value for `editorValue`
+ */
+export interface IUpdateEditorValue extends Action {
+  type: string
```

Comment:
This should be redundant, all `Action` is expected to have a type field, please verify